src/databricks/labs/ucx/runtime.py

```diff
@@ -581,6 +584,35 @@ def scan_tables_in_mounts_experimental(
     ).snapshot()
 
 
+@task("all")
+def parse_logs(cfg: WorkspaceConfig, ws: WorkspaceClient, sql_backend: SqlBackend, _install: Installation):
+    """Parse the logs after workflow completion."""
+    _ = ws
+
+    task_name_format = r"^(\w+)\.log"
+    task_name_pattern = re.compile(task_name_format)
+
+    logs_recorder = LogsRecorder(
+        _install.install_folder(),
+        # Workaround by setting attributes in `run_task` to retrieve workflow ids
+        cfg.dynamically_set_attribut__workflow,  # type: ignore
+        cfg.dynamically_set_attribut__job_id,  # type: ignore
+        cfg.dynamically_set_attribut__workflow_run_id,  # type: ignore
+        sql_backend,
+        cfg.inventory_database,
+    )
+
+    for log_file in logs_recorder.log_path.glob("*.log"):
```

Comment:
this is a bit incorrect glob pattern. in `TaskLogger` set the interval to 1 minute and run the assessment workflow to see what i mean.